### PR TITLE
YD-686 Added length check for activity comments

### PR DESCRIPTION
**Pull request by Bert-R**
Earlier, there was no explicit check. Testing proved I could make a comment with 237 *s. In reality, this limit might vary due to the encryption. To stay on the safe side and still give plenty of room for a comment, the maximum is now set to 200 characters.

Along with this:
* Corrected a few error messages by taking out double curly braces
* Removed two duplicated error messages

### CODE REVIEW

File: appservice/src/intTest/groovy/nu/yona/server/ActivityCommentTest.groovy [hunk 1]
```diff
@@ -430,6 +432,45 @@ class ActivityCommentTest extends AbstractAppServiceIntegrationTest
...
+		assert budgetGoalNewsBuddyBob
+		def responseOverviewsBobAsBuddyAll = appService.getDayActivityOverviews(richard, richard.buddies[0], ["size": 14])
+		assertResponseStatusOk(responseOverviewsBobAsBuddyAll)
+		def responseDetailsBobAsBuddy = appService.getDayDetailsFromOverview(responseOverviewsBobAsBuddyAll, richard, budgetGoalNewsBuddyBob, 1, "Tue")
+		assertResponseStatusOk(responseDetailsBobAsBuddy)
+		def messageText = StringUtils.repeat("*", messageLength)
+		def messageJson = """{"message": "$messageText"}"""
+
+		then:
+		def response = appService.yonaServer.createResourceWithPassword(responseDetailsBobAsBuddy.responseData._links."yona:addComment".href, messageJson, richard.password)
```
Comment (aristotelos) [inline appservice/src/intTest/groovy/nu/yona/server/ActivityCommentTest.groovy:460]: We may extract a method `addComment` here

File: core/src/main/resources/messages/messages.properties [hunk 2]
```diff
@@ -28,14 +28,13 @@ notification.message.body=Tap to open message
 error.invalid.request=Invalid request
 error.invalid.uuid=String ''{0}'' is not a valid UUID
 error.invalid.date=String ''{0}'' is not a valid date
-error.invalid.date.too.early=Date ''{0}}'' is too early. Earliest possible date is ``{1}}``
+error.invalid.date.too.early=Date ''{0}'' is too early. Earliest possible date is ``{1}``
```
Comment (aristotelos) [inline core/src/main/resources/messages/messages.properties:31]: Good catch... Another one: I see two ticks `''` vs. two back-ticks ````, which we may correct too

**Review by aristotelos — COMMENTED** [review]
Looks great, added a suggestion

File: appservice/src/intTest/groovy/nu/yona/server/ActivityCommentTest.groovy (hunk 1 above)
Comment (Bert-R) [reply appservice/src/intTest/groovy/nu/yona/server/ActivityCommentTest.groovy]: Good point. Done

File: core/src/main/resources/messages/messages.properties (hunk 2 above)
Comment (Bert-R) [reply core/src/main/resources/messages/messages.properties]: There were a few more of these. Replaced them all

**Comment by Bert-R** [comment]
Pushed an update

**Review by aristotelos — APPROVED** [review]

**Pull request merged**